code/modules/mob/living/simple_animal/hostile/skeleton.dm

```diff
@@ -122,3 +121,163 @@
 /mob/living/simple_animal/hostile/skeleton/plasmaminer/Initialize()
 	. = ..()
 	set_light(2)
+	
+/mob/living/simple_animal/hostile/skeleton/robust
+	name = "robust skeleton"
+	desc = "A real bonefied skeleton, and this one seems to have a bone to pick with you."
+	del_on_death = 0
+	icon_dead = null
+	loot = list()
+	gold_core_spawnable = HOSTILE_SPAWN
+	var/datum/action/innate/skeleton/robust/grave_warden/GW
+	var/gravewardened = 0
+	
+/mob/living/simple_animal/hostile/skeleton/robust/Initialize()
+	. = ..()
+	GW = new
+	GW.Grant(src)
+
+/mob/living/simple_animal/hostile/skeleton/robust/AttackingTarget()
+	if(istype(target, /obj/vehicle/sealed/car/grave_warden))
+		return
+	. = ..()
+	if(. && ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(H.stat == DEAD)
+			convert(H)
+
+/mob/living/simple_animal/hostile/skeleton/robust/death(gibbed)
+	var/obj/effect/decal/remains/human/skelebones = new (src.loc)
+	..()
+	if(health > 0)
+		return
+	else
+		if(!gibbed && skelebones.loc)
+			sleep(300)
+			if(skelebones.loc)
+				INVOKE_ASYNC(src, .proc/revive, 1, 1)
+				src.loc = skelebones.loc
+				qdel(skelebones)
+				visible_message(
+					"<span class='danger'>[src] re-forms from the pile of bones!</span>",
+					"<span class='userdanger'>You rise again!</span>")
+			else
+				to_chat(src, "<span class='notice'>Without your bones, your spirit eases up and ascends.</span>")
+				qdel(src)
+		else
+			to_chat(src, "<span class='notice'>Without your bones, your spirit eases up and ascends.</span>")
+			qdel(src)
+			
+/mob/living/simple_animal/hostile/skeleton/robust/proc/convert(mob/living/carbon/L)
+	if(!L)
+		return
+	visible_message(
+		"<span class='danger'>[src] rips out [L]'s skeleton!</span>",
+		"<span class='userdanger'>You free the skeleton from inside [L]!</span>")
+	var/mob/living/simple_animal/hostile/skeleton/robust/newskele = new (L.loc)
+	newskele.faction = faction.Copy()
+	if(L.mind)
+		L.mind.transfer_to(newskele)
+	L.gib()
+
+/obj/vehicle/sealed/car/grave_warden
+	name = "grave warden"
+	desc = "A pile of skeletons somehow forming one large skeleton."
+	icon = 'icons/mob/lavaland/lavaland_monsters.dmi'
+	icon_state = "legion"
```

Comment:
you don't have to do my suggestion of a skeleton ball but reusing the sprite of a common lavaland mob is a really bad idea

Reply:
Considering it's sized up, it's pretty obvious it's not a legion.  As I cannot make sprites, I'll have to figure out something.  I really like the DS3 skeletonball though, so if I can, I'd want that.

Reply:
Also, it plays Megalovania sounds when it moves.